external/fv3fit/setup.py

```diff
@@ -11,7 +11,7 @@
     "scikit-learn>=0.22",
     "fsspec>=0.6.2",
     "pyyaml>=5.1.2",
-    "tensorflow>=2.2.0",
+    "tensorflow==2.5.2",
```

Comment:
Needed to match CUDA library compatibility in the chosen image.

Reply:
Maybe add a comment saying this?